packages/flutter_tools/lib/src/base/file_system.dart

```diff
@@ -65,34 +65,12 @@ class FileSystemUtils {
     bool shouldCopyFile(File srcFile, File destFile),
```

Comment:
> make FileSystemUtils.copyDirectorySync delegate to copyDirectory, to avoid a large diff/breaking change

I don't mind reviewing large diffs to migrate existing `FileSystemUtils.copyDirectorySync` usages to just `copyDirectory`.  What breaking changes are you concerned about (ono is this used in g3?)

Reply:
Done, just mostly dreading fixing the tests that mock it, but there weren't too many.